src/tests/functional/plugin/conformance/test_runner/api_conformance_runner/src/behavior/ov_plugin/properties.cpp

```diff
@@ -16,9 +16,7 @@ const std::vector<ov::AnyMap> inproperties = {
         {ov::device::id("UNSUPPORTED_DEVICE_ID_STRING")},
 };
 
-const std::vector<ov::AnyMap> auto_batch_inproperties = {
-        {{ov::auto_batch_timeout(-1)}},
```

Comment:
remove the test with ov::auto_batch_timeout(-1), cause the type of this variable is unsigned int. The original code can work is because the -1 will be converted to 4294967295, and 4294967295 exceed the limitation of std::stoi() which throw exception. The behavior is not expected.

Reply:
Add related discussion:
https://github.com/openvinotoolkit/openvino/pull/15991/files#r1147975659